app/views/hyrax/file_sets/media_display/_image.html.erb

```diff
@@ -4,14 +4,13 @@
       <%= image_tag thumbnail_url(file_set),
                     class: "representative-media",
                     alt: "",
-                    role: "presentation" %>
-      <%= link_to hyrax.download_path(file_set),
+                    role: "presentation",
+                    width: 250 %>
```

Comment:
Please move this to stylesheets where it'll be easier for downstream consumer to override.